### ISSUE TITLE
Redirection Error

### ISSUE THREAD

**Issue by jimmyessbee** [johngodley/redirection#731]
I'm not sure what happened. There were no new plugins. It may have been a OS update that caused the issue, but I dno't check this often and then suddenly it didn't work. Redirection still works, I simply cannot configure it. 

```
Plugin: 3.1.1
WordPress: 4.9.2 (single)
PHP: 7.0.22-0ubuntu0.16.04.1
Browser: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:58.0) Gecko/20100101 Firefox/58.0
REST API: http://www.montpelier.k12.nd.us/index.php/wp-json/

Error: No route was found matching the URL and request method
Action: redirection/v1/redirect?orderby=id GET
Code: 404 Not Found
Raw: {"code":"rest_no_route","message":"No route was found matching the URL and request method","data":{"status":404}}

Error: No route was found matching the URL and request method
Action: redirection/v1/group GET
Code: 404 Not Found
Raw: {"code":"rest_no_route","message":"No route was found matching the URL and request method","data":{"status":404}}
```

**Comment by johngodley** [comment]
Can you try the 3.2 beta and let me know how it goes?

https://github.com/johngodley/redirection/releases/tag/3.2-beta

**Comment by jimmyessbee** [comment]
When I went out to install this, I saw that there was an update listed for my plug-in which was 3.2. It didn’t call it a beta anymore, but I installed it to no avail. I thought I would then download the beta from the link you shared, but when I do that, it will not let me upload and install – because the directory already exists. I could delete that directory, but I’ve got a lot of redirections and I do hate the thought of losing them .

I also did go through and deactivate all other plugins and still had the same problem.

**Comment by johngodley** [comment]
Yes, I released 3.2. There's no need to try the beta.

Look at http://www.montpelier.k12.nd.us/index.php/wp-json/ I can see that Redirection isn't listed and neither are any other API routes. Do you have a plugin running that disables the REST API? If so you will need to allow Redirection routes

**Comment by jimmyessbee** [comment]
Thank you.

Interestingly enough, the reason it was broken was the very reason I had the redirection tool installed in the first place.

My files do not sit in the html directory, but rather in the wordpress directory within the default html directory. But, I have wordpress directing all traffic to the main directory.

So, when you had me follow that URL, I knew it wouldn’t hold anything, because it would have to go here:

http://www.montpelier.k12.nd.us/wordpress/index.php/wp-json/

I’m not sure what changed that broke it, but I just started playing with the path for REST API and got it to work.

Thank you.
